ws.c

```diff
@@ -428,9 +453,13 @@ make_ws_frame(struct evbuffer *output, enum WebSocketFrameType frame_type,
 		header[pos++] = (len >> 8) & 0xFF; /* rightmost first */
 		header[pos++] = len & 0xFF;
 	} else {				 /* >2^16-1 */
-		header[pos++] = 127; /* 64 bit length */
-
-		pos += 8;
+		int i;
+		const uint64_t tmp64 = len;
+		header[pos++] = 127;            /* 64 bit length */
+		/* swab bytes from host byte order to big endian */
```

Comment:
Same, what if we are on a big-endian system?

Reply:
It's a no-op there.

Although it does eat cycles.

Do you have a compiler-define that defines big endian? I can `#ifdef` the code out then if the CPU cycle eating for a no-op is an issue for you on big endian.

Reply:
I just checked there is no project-wide `#define`  regarding endian-ness, nor is there anything in `event2/event-config.h`.  The closest thing is a per-file compile options applied to `sha1.c` that defines `LITTLE_ENDIAN=1` or `BIG_ENDIAN=1` which is required by that file to compile.

So there is no way to test endianness from the sources -- not without modification of `CMakeLists.txt` and/or the autoconf build system.

I can add such a modification to this PR if you like, and then `#ifdef` out this code on big-endian.  Or we can leave it as-is (since it's harmless) and do the work in a follow-up PR to add this to the build system for `ws.c`, etc.

Reply:
It is OK to keep this code as is (but if you will replace some constant in the decoding loop, then let's replace them here as well).

Reply:
Hey sorry for belated reply -- I got caught up in some other commitments. I will update this PR this weekend hopefully or sometime next week worst case, I hope!